.github/workflows/python-conda-build.yaml

```diff
@@ -25,13 +25,12 @@ jobs:
           export PATH="$GITHUB_PATH:$PATH"
           source activate scAR
           scar --help
-      # - name: Lint with flake8
-      #   run: |
-      #     conda install flake8
-      #     # stop the build if there are Python syntax errors or undefined names
-      #     flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-      #     # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-      #     flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+      - name: Run pylint
+        run: |
+          export PATH="$GITHUB_PATH:$PATH"
+          source activate scAR
+          conda install pylint
+          pylint scAR --fail-under 5
```

Comment:
The pylint github action fails if the score is below 5. There are still a lot of lint issues but we can fix them over time. @CaibinSh  I think this would be something that you can spend some time.